scripted_interpreted/arnoldc/README.md

```diff
@@ -0,0 +1,8 @@
+Run in command line
+
+//if you havent generated class file
+>java -jar ArnoldC.jar hello.arnoldc
```

Comment:
Could provide a link to get the ArnoldC.jar file, such as `wget http://lhartikk.github.io/ArnoldC.jar`